docs/src/extend/custom-processors.md

```diff
@@ -140,7 +140,83 @@ By default, ESLint does not perform autofixes when a custom processor is used, e
 
 You can have both rules and custom processors in a single plugin. You can also have multiple processors in one plugin. To support multiple extensions, add each one to the `processors` element and point them to the same object.
 
-**The `meta` object** helps ESLint cache the processor and provide more friendly debug message. The `meta.name` property should match the processor name and the `meta.version` property should match the npm package version for your processors. The easiest way to accomplish this is by reading this information from your `package.json`.
+### Clarifying the Use of Meta Objects:
+
+#### Plugin Meta-Object
+
+The top-level meta object of the plugin provides essential metadata about the plugin itself. This meta object is used when the processor is specified in the format plugin-name/processor-name. This helps ESLint to cache the plugin and provide more informative debug messages.
+
+Example:
+
+```js
+// eslint.config.js
+import markdown from "eslint-plugin-markdown";
+
+export default [
+    {
+        files: ["**/*.md"],
+        plugins: {
+            markdown
+        },
+        processor: "markdown/markdown"
+    },
+    // ... other configs
+];
+```
+
```

Comment:
We should add some content here explaining what the processor name is interpreted as.